python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -144,3 +145,82 @@ def run(
                 metricName, np.mean(np.abs(result["corr"])), extras=extras
             )
         )
+
+
+class FluxStatisticConfig(MeasurementTaskConfig):
+    """Class to configure the flux statistic measurement task.
+    """
+
+    columns = DictField(
```

Comment:
I like the detailed doc sting here. Was there a reason to define `columns` and `columnsBand` again here when they are implemented in `MeasurementTaskConfig`?

Reply:
A good point - this was originally copied over from the above `TExTableConfig`, and I modified the doc-strings to be more complete to my eye. I hadn't realized they were already implemented in `MeasurementTaskConfig`. If you like, I could edit the doc strings in `ConfigBase.py` on this ticket, and remove the configs here in `FluxStatisticConfig`?

Reply:
Thanks, that sounds excellent.

Reply:
Will do. As an aside, I probably won't edit the above `TExTableConfig` here to remove these config options, as I'm not familiar enough with that task to perform the do-no-harm checks that I'd want.

Reply:
The idea is that the individual measurement task configs can override `columns` and `columnsBand` to specify the columns needed for a specific metric measurement. In the case of `TExTableConfig`, we wanted to specify a set of default columns needed for the ellipticity residual measurement. In the case of the `FluxStatisticConfig`, it seems that the columns will be specified in the yaml pipeline files rather than to have defaults set since it seems there isn't a particular column that would be the default.

Reply:
I was confused with the above `TExTableConfig`, as `columnsBand` seems to be doing nothing special to require an override here, hence perhaps warranting its removal? But, that's beyond the scope of this ticket.